bin/tictactoe

```diff
@@ -1,3 +1,38 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+
+puts "Welcome to Tic Tac Toe!"
+
+
+def start_game
+  puts "How many users to play the game?"
+  puts "1 player"
+  puts "2 players"
+  puts "Or ZERO player, meaning computer vs computer."
+  input=gets.strip
+
+  if input=="1"
+    board=Board.new
+    game=Game.new(player_1=Player::Human.new("X"), player_2=Player::Computer.new("O"))
+    game.play
+  
+  elsif input=="2"
+    board=Board.new
+    game=Game.new(player_1=Player::Human.new("X"), player_2=Player::Human.new("O"))
+    game.play
+  elsif input=="ZERO" || "0"
+    board=Board.new
+    game=Game.new(player_1=Player::Computer.new("X"), player_2=Player::Computer.new("O"))
+    game.play
+  else
+    puts "You entered an invalid number, please pick again"
+    start_game
+  end
+end
```

Comment:
Prefer to encapsulate CLI logic into a class. For example, the only code in your bin executable should be:

```ruby
#!/usr/bin/env ruby
       
require_relative '../config/environment'

TicTacToeCLI.new.call
```

And then you'd have a file: `lib/tic_tac_toe_cli.rb` with:

```ruby
class TicTacToeCLI
  def initialize
  end

  def call
   # Your game logic
  end
end
```

The code in your bin would generally just go in `call`.